### DISCUSSION TITLE
Doc comment on FAQ (/more-resources/faq)

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#1032] [Docs/blog comments]
# Doc comment on FAQ (/more-resources/faq)

General

https://weaviate.io/developers/weaviate/more-resources/faq

**Comment by adebayo-pixel** [comment]
can someone tell me about Weaviate in terms of : Performance, Existing MLOps, developer experience, Reliability and Cost

**Comment by kunal8164705** [comment]
is it possible to create a multivector field index (i.e. for each field in index there must be seperate embedding).
for example i have 
```
{
"name":"F7",
"genere":"action"
}

### now i want to know is this possible?
### seperate vector for different field ?
{
"name":"F7",
"name_vector":[-1.2,....],

"genere":"action",
"genre_vector":[-1.243.....]
}
```

**Reply by dudanogueira** [reply]
Hi! This is not supported right now.
There is a proposal for this feature here: https://github.com/weaviate/weaviate/issues/3501
Thumbs up there is appreciated ;)

**Comment by prandell** [comment]
Are we able to see calls made to openai using text2vec-openai? I want to see the result of nearText's vectorisation

**Reply by dudanogueira** [reply]
Hi! This is not supported for now.

**Comment by YixingWang1999** [comment]
Will more replicas and more sharding(more pods) both help the performance of search a nearest vector?

**Reply by dudanogueira** [reply]
Hi! adding replicas/shards will increase your query throughput, according to here: https://weaviate.io/developers/weaviate/concepts/cluster#motivation-2-higher-query-throughput

In order to deal with performance, you will need to experiment with different Indexing Configurations, that will depend on your needs, as stated here:
https://weaviate.io/developers/weaviate/configuration/indexes#how-to-configure-hnsw

Another feature that can help you on that is Product Quantization:
https://weaviate.io/developers/weaviate/concepts/vector-index#hnsw-with-product-quantization-pq

**Comment by kameshwaran-s** [comment]
What type of data (besides text) are supported in Weaviate?
Is it possible to store and search for vector embeddings of time series data?

**Comment by sri-postgres-cassandra** [comment]
I am new to weaviate and would need help to configure weaviate on a VM. can someone please share me the steps to install weaviate in a VM and steps to setup a cluster . I searched for those did not find them .

**Reply by dudanogueira** [reply]
Hi! If you plan on deploying using Docker, a good start is to use our [docker configurator tool](https://weaviate.io/developers/weaviate/installation/docker-compose#configurator)

You can also use [Kubernetes](https://weaviate.io/developers/weaviate/installation/kubernetes)

Also, [Resource Planning](https://weaviate.io/developers/weaviate/concepts/resources) is a also a good documentation to better size your deployment.

Let me know if this helps!

**Reply by sri-postgres-cassandra** [reply]
Thank you Very much for the help. Can you please let me know if Weaviate can be configured on a VM and can we configure cluster ? I don't see any steps to install Weaviate on a VM and cluster setup .

**Comment by jamesschinnerplxs** [comment]
I'm using the python client (v4) and have a class schema defined, Is there an existing method to check for schema mismatches? Ie, the collection is created once from the defined table, but some one modified the schema later on in code, how would I detect such a situation (before some other failure)?

**Comment by FalconDinesh** [comment]
Can I set a property value to be unique? like email should be unique in all the documents
